scannerpython.properties

```diff
@@ -0,0 +1,40 @@
+category=Scanners
+name=SonarScanner for Python
+license=GNU LGPL 3
+organization=SonarSource
+organizationUrl=https://www.sonarsource.com
+homepageUrl=https://docs.sonarsource.com/sonarqube-server/latest/analyzing-source-code/scanners/sonarscanner-for-python/
+issueTrackerUrl=https://sonarsource.atlassian.net/jira/software/c/projects/SCANPY/issues
+sourcesUrl=https://github.com/SonarSource/sonar-scanner-python
+archivedVersions=0.1.0.285,0.1.0.340,0.2.0.520,1.0.0.1453,1.0.1.1548
+publicVersions=1.0.2.1722
+
+1.0.2.1722.description=Fix incompatibility with tarfile.extractall
+1.0.2.1722.date=2025-05-28
+1.0.2.1722.changelogUrl=https://sonarsource.atlassian.net/issues/?jql=project%20%3D%20%22Sonar%20Scanner%20Python%22%20AND%20fixVersion%20%3D%201.0.2
+1.0.2.1722.downloadUrl=https://pypi.org/project/pysonar/#history
```

Comment:
Instead of a link to history, can we maybe provide the link to the specific version on pypi.org?
For instance: `https://pypi.org/project/pysonar/1.0.2.1722/`